composer.json

```diff
@@ -9,7 +9,8 @@
         "barryvdh/laravel-debugbar": "~2.4",
         "laravel/framework": "5.4.*",
         "laravel/tinker": "~1.0",
-        "tom-lingham/searchy" : "2.*"
+        "maatwebsite/excel": "~2.1.0",
```

Comment:
So sad we are sticking with it... I get that we don't want to start something new.

Reply:
Not sure I follow. not keen on Maatwebsite/excel?

Reply:
I thought at some point we'd wished we could give up on them... or had to freeze the dependency... but I just checked and I think it's OK.

I  think the issue was we can't test excel sheets because headers already sent. But ignore me. :)